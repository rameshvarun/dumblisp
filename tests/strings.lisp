(set hello "Hello")
(set world "World")

(set helloworld (strcat (strcat hello " ") world))

(print helloworld)
(print (strlen hello))
(print (substr helloworld 11 0))

(print (strcmp hello "Hello"))
(print (strcmp hello world))