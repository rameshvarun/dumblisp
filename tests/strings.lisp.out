Hello World
5

0
-15
